File: javaMasai/unit-4/sprint-4/day-1/you/csbt/Csbt.java


public class Csbt {
    public static int reverse(int num) {
        int reversed = 0;
        while (num != 0) {
            int digit = num % 10;
            reversed = reversed * 10 + digit;
            num /= 10;
        }
        return reversed;
    }

    public static void main(String[] args) {
        int num = 1211; // Replace with any 3 digit number
        int reversedNum = reverse(num);

        if (num == reversedNum) {
            System.out.println(num + " is a palindrome");
        } else {
            System.out.println(num+" is not a palindrome");
        }
    }
}

File: javaMasai/unit-4/sprint-4/day-1/you/csbt/csbt.js
function reverse(num) {
    return parseInt(num.toString().split("").reverse().join(""));
}


let num = 13212; // Replace with any 3 digit number
let reversedNum = reverse(num);
console.log(typeof reversedNum)

console.log(reversedNum)
if (num === reversedNum) {
    console.log(`${num} is a palindrome`);
} else {
    console.log(`${num} is not a palindrome`);
}
